client-python/grakn/exception/ClientError.py

```diff
@@ -0,0 +1,21 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#   http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+class ClientError(Exception):
```

Comment:
Is there a special reason for why we're naming this class `ClientError`? If this class is extending `Exception`, in order to represent exceptions in the `Grakn` client driver, so that we can inform the user about an exception from their `Grakn` driver, perhaps we should call it `GraknException`? @flyingsilverfin @lolski

Reply:
Python convention is `*Error` which I will rename to: `GraknError`